website/pages/docs/integrations/vault-integration.mdx

```diff
@@ -225,485 +151,319 @@ disallowed policies group. An example of this is shown below:
+```shell-session
+# Create a namespace "engineering" within Vault
+$ vault namespace create engineering
 
-```text
-Success! Data written to: database/roles/accessdb
+# Create a child namespace "frontend" under "engineering"
+$ vault namespace create -namespace=engineering frontend
 ```
 
-### Step 12: Generate PostgreSQL Credentials
+### Required Vault Policies
 
-You should now be able to generate dynamic credentials to access your database.
-Run the following command to generate a set of credentials:
+Policies are configured per Namespace. We will apply the policy in the example above to each Namespace.
 
 ```shell-session
-$ vault read database/creds/accessdb
-```
-
-The previous command should return output similar to what is shown below:
+# Apply the policy to our engineering namespace
+$ vault policy write -namespace=engineering nomad-server nomad-server-policy.hcl
 
-```text
-Key                Value
----                -----
-lease_id           database/creds/accessdb/3JozEMSMqw0vHHhvla15sKTW
-lease_duration     1h
-lease_renewable    true
-password           A1a-3pMGjpDXHZ2Qzuf7
-username           v-root-accessdb-5LA65urB4daA8KYy2xku-1542318363
+# Apply the policy to our engineering/frontend namespace
+$ vault policy write -namespace=engineering/frontend nomad-server nomad-server-policy.hcl
 ```
 
-Congratulations! You have configured Vault's connection to your database and can
-now generate credentials with the previously specified privileges. Now we need
-to deploy our application and make sure that it will be able to communicate with
-Vault and obtain the credentials as well.
+We will also configure the previously configured `nomad-cluster` role with each Namespace
 
-### Step 13: Create the `access-tables` Policy for Your Nomad Job to Use
+```shell-session
+# Create the token role with Vaule for engineering
+$ vault write -namespace=engineering /auth/token/roles/nomad-cluster @nomad-cluster-role.json
 
-Recall from [Step 5][step-5] that we specified a policy named `access-tables` in
-our `allowed_policies` section of the token role. We will create this policy now
-and give it the capability to read from the `database/creds/accessdb` endpoint
-(the same endpoint we read from in the previous step to generate credentials for
-our database). We will then specify this policy in our Nomad job which will
-allow it to retrieve credentials for itself to access the database.
+# Create the token role with Vaule for engineering/frontend
+$ vault write -namespace=engineering/frontend /auth/token/roles/nomad-cluster @nomad-cluster-role.json
+```
 
-On the Nomad server (which is also running Vault), create a file named
-`access-tables-policy.hcl` with the following content:
+The [Nomad agent Vault integration][config] congfiguration supports specifying a Vault Namespace, but since
+we will be using multiple and have configured our policies it can be left blank so by default Nomad interacts with Vault in it's root Namespace.
```

Comment:
The sentence is a bit unwieldy... not a big deal, but here's an attempt at making it read easier:

```suggestion
we will be using multiple it can be left blank. By default Nomad will interact with Vault's root Namespace, but individual jobs may specify other Vault Namespaces to use.
```